### PR TITLE
fix: removing Filtering for exit calls cell in trace table

### PR DESCRIPTION
**Pull request by itssharmasandeep**
## Description
Removing Filtering for exit calls cell in trace table

### Testing
Local testing done

### Checklist:
- [x] My changes generate no new warnings

### CODE REVIEW

File: projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts [hunk 1]
```diff
@@ -52,10 +49,10 @@ interface CellData {
   parser: CoreTableCellParserType.NoOp
 })
 export class ExitCallsTableCellRendererComponent extends TableCellRendererBase<CellData, Trace> implements OnInit {
-  public readonly apiCalleeNameCount: string[][];
+  public readonly apiCalleeNameEntires: [string, string][];
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts:52]:
```suggestion
  public readonly apiCalleeNameEntries: [string, string][];
```

(and references)

File: projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts [hunk 2]
```diff
@@ -52,10 +49,10 @@ interface CellData {
...
   public readonly apiExitCalls: number;
-  public readonly maxShowApiCalleeNameCount: number = 10;
-  public readonly totalCountOfDifferentApiCallee!: number;
+  public readonly MAX_API_CALLEE_TO_SHOW: number = 10;
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts:54]: With a const like this, we should make it static (and lift it above the instance vars)

File: projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts [hunk 3]
```diff
@@ -52,10 +49,10 @@ interface CellData {
...
+  public readonly uniqueApiCallee: number;
```
Comment (arjunlalb) [inline projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts:58]: I'd recommend `uniqueApiCalleeCount`

File: projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts (hunk 2 above)
Comment (arjunlalb) [reply projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.ts]: also, please rename to `MAX_API_CALLEES_TO_SHOW`

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**